src/docs/product/performance/web-vitals.mdx

```diff
@@ -0,0 +1,58 @@
+
+If you see a region of interest on any of the histograms, click and drag over the area to zoom in for a more detailed view. You may also want to see more information related to the transactions in the histograms. Click "Open in Discover" beneath the web vital of choice to build a custom query to investigate further. For more details, see the full documentation for the Discover [Query Builder](/product/discover-queries/query-builder/).
+
+## Vitals
+
+### First Paint (FP)
+
+First Paint (FP) measures the time for the first pixel to appear in the viewport, rendering any visual change from what was previously displayed. This may be in any form from the document object model (DOM) such as background color, canvas, image, etc. It helps developers understand if anything is happening to render the page.
+
+### First Contentful Paint (FCP)
+
+First Contentful Paint measures the time for the first content to render in the viewport. This may be in any form from the document object model (DOM) such as images, svgs, text blocks, etc. FCP frequently overlaps with First Paint (FP). It helps developers understand how long it takes before the user begins to perceive any content on the page.
+
+### Largest Contentful Paint (LCP)
+
+Largest Contentful Paint measures the render time for the largest content to appear in the viewport. This may be in any form from the document object model (DOM) such as images, svgs, text blocks, etc. It is the largest pixel area in the viewport, thus most visually defining. It helps developers understand how long it takes before the user perceives the main content on the page.
+
+### First Input Delay (FID)
+
+First Input Delay measures the response time when the user tries to interact with the viewport. Actions may include clicking a button, link or other custom Javascript controller. It is key in helping the user determine if a page is usable or not.
+
+### Cumulative Layout Shift (CLS)
+
+Cumulative Layout Shift is the sum of individual layout shift scores for every unexpected element shift that occurs during the rendering process. Imagine navigating to an article and trying to click a link before the page finishes loading. Before your cursor even gets there, the link may have shifted down due to an image rendering. Rather than using duration for this web vital, the score represents how disruptive and visually unstable the shifts are.
+
+Each layout shift score is calculated using an impact and distance fraction where impact fraction is the total visible area that the element affects between the two rendered frames and distance fraction measures the distance it has moved relative to the viewport.
+
+Layout shift score = Impact Fraction * Distance Fraction
+
+Let’s take a look at the example above which has one unstable element - the body text. The impact fraction is roughly 50% of the page and moves the body text down by 20%. The layout shift score is 0.1, the product of 0.5*0.2. Thus, CLS is 0.1.
```

Comment:
I think we're missing this exact example above.

Reply:
Yes, I haven't added the necessary screenshots yet. I'm tracking them in the TODO in the description above and will be checking them off as they're added in 😄

Reply:
Great. Thanks, Tony! 🌻